src/xmipp/libraries/data/ctf.h

```diff
@@ -55,6 +55,16 @@ const MDLabel CTF_ALL_LABELS[] =
         MDL_CTF_K
     };
 
+/** Convert a site_t value to double **/
+inline double convertSizeTToDouble(size_t data)
```

Comment:
This function would be better at XmippCore utils

Reply:
You're right. I was already saying to David that I had to move it, because it was not the right place, in fact, I think that in the end I would eliminate it.